diff --git a/node_modules/react-native-background-timer/index.js b/node_modules/react-native-background-timer/index.js
index f64e4b6..fe99508 100644
--- a/node_modules/react-native-background-timer/index.js
+++ b/node_modules/react-native-background-timer/index.js
@@ -12,14 +12,15 @@ const Emitter = new NativeEventEmitter(RNBackgroundTimer);
 class BackgroundTimer {
   constructor() {
     this.uniqueId = 0;
-    this.callbacks = {};
+    this.callbacks = new Map();
 
     Emitter.addListener('backgroundTimer.timeout', (id) => {
-      if (this.callbacks[id]) {
-        const callbackById = this.callbacks[id];
-        const { callback } = callbackById;
-        if (!this.callbacks[id].interval) {
-          delete this.callbacks[id];
+      if (this.callbacks.has(id)) {
+        const callbackById = this.callbacks.get(id);
+
+        const { callback, interval, timeout } = callbackById;
+        if (!interval) {
+          this.callbacks.delete(id);
         } else {
           RNBackgroundTimer.setTimeout(id, this.callbacks[id].timeout);
         }
@@ -68,18 +69,18 @@ class BackgroundTimer {
   setTimeout(callback, timeout) {
     this.uniqueId += 1;
     const timeoutId = this.uniqueId;
-    this.callbacks[timeoutId] = {
+    this.callbacks.set(timeoutId, {
       callback,
       interval: false,
       timeout,
-    };
+    });
     RNBackgroundTimer.setTimeout(timeoutId, timeout);
     return timeoutId;
   }
 
   clearTimeout(timeoutId) {
-    if (this.callbacks[timeoutId]) {
-      delete this.callbacks[timeoutId];
+    if (this.callbacks.has(timeoutId)) {
+      this.callbacks.delete(timeoutId);
       // RNBackgroundTimer.clearTimeout(timeoutId);
     }
   }
@@ -87,18 +88,18 @@ class BackgroundTimer {
   setInterval(callback, timeout) {
     this.uniqueId += 1;
     const intervalId = this.uniqueId;
-    this.callbacks[intervalId] = {
+    this.callbacks.set(intervalId, {
       callback,
       interval: true,
       timeout,
-    };
+    });
     RNBackgroundTimer.setTimeout(intervalId, timeout);
     return intervalId;
   }
 
   clearInterval(intervalId) {
-    if (this.callbacks[intervalId]) {
-      delete this.callbacks[intervalId];
+    if (this.callbacks.has(intervalId)) {
+      this.callbacks.delete(intervalId);
       // RNBackgroundTimer.clearTimeout(intervalId);
     }
   }
